src/org/zaproxy/zap/extension/alert/PopupMenuAlertBugTracker.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ * 
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ * 
+ * Copyright 2010 psiinon@gmail.com
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License"); 
+ * you may not use this file except in compliance with the License. 
+ * You may obtain a copy of the License at 
+ * 
+ *   http://www.apache.org/licenses/LICENSE-2.0 
+ *   
+ * Unless required by applicable law or agreed to in writing, software 
+ * distributed under the License is distributed on an "AS IS" BASIS, 
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. 
+ * See the License for the specific language governing permissions and 
+ * limitations under the License. 
+ */
+
+package org.zaproxy.zap.extension.alert;
+
+import java.awt.Dimension;
+
+import javax.swing.JTree;
+import javax.swing.tree.DefaultMutableTreeNode;
+import javax.swing.tree.TreePath;
+
+import org.parosproxy.paros.extension.ExtensionPopupMenuItem;
+import org.parosproxy.paros.core.scanner.Alert;
+import org.zaproxy.zap.extension.alert.AlertPanel;
+import org.zaproxy.zap.extension.alert.ExtensionAlert;
+
+
+public class PopupMenuAlertBugTracker extends ExtensionPopupMenuItem {
+
+	private static final long serialVersionUID = 1L;
+
+
+	public PopupMenuAlertBugTracker() {
+		super();
+	}
+
+	public PopupMenuAlertBugTracker(String label) {
+		super(label);
+	}
+
+    public Alert[] getAlertNodes(ExtensionAlert extAlert) {
```

Comment:
It should be an abstract class that calls the action methods instead (it reduces the boiler plate code in extending classes, as they just need to implement the corresponding actions). In that case this method does not need to be public.